client/ClientApp.cpp

```diff
@@ -103,16 +103,12 @@ const std::map<int, PlayerInfo>& ClientApp::Players() const
 std::map<int, PlayerInfo>& ClientApp::Players()
 { return m_player_info; }
 
-const std::map<int, Message::PlayerStatus>& ClientApp::EmpireStatus() const
-{ return m_empire_status; }
-
-std::map<int, Message::PlayerStatus>& ClientApp::EmpireStatus()
-{ return m_empire_status; }
-
 void ClientApp::SetEmpireStatus(int empire_id, Message::PlayerStatus status) {
     if (empire_id == ALL_EMPIRES)
```

Comment:
can also remove this check since `ALL_EMPIRES` should never be a valid empire id